test/unit/spec/EnrollChoices_spec.js

```diff
@@ -186,24 +199,38 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
       return setup(res);
     }
 
-    function setupMultipleFactorEnrollments (factorName, status, enrollment, policyEnrollment, singleFactorRes) {
-      var res = singleFactorRes ? deepClone(singleFactorRes) : deepClone(resAllFactors);
-      var index = singleFactorRes ? 0 : factorEnrollList[factorName].index;
-      res.response._embedded.factors[index].status = status;
-      res.response._embedded.factors[index].enrollment = enrollment;
-      res.response._embedded.factors[index].policy.enrollment = policyEnrollment;
-      return setup(res);
+    function setupMultipleFactorEnrollments (options) {
+      var res = options.singleFactorRes ? deepClone(options.singleFactorRes) : 
+        options.useProfiles ? deepClone(resAllFactorsProfile) : deepClone(resAllFactors);
+      var index = options.singleFactorRes ? 0 : factorEnrollList[options.factorName].index;
+      var factor = res.response._embedded.factors[index];
+      factor.status = options.status;
+      factor.enrollment = options.enrollment;
+      if (options.useProfiles) {
+        var cardinality = {
```

Comment:
can we move this logic to a function? Something like
```
factor.profiles[0] = createProfile(options.cardinality);
```

or
```
setProfileData(factor.profiles[0], options.cardinality);
```

I'm guessing in the future we will need to add more things to test the profiles, so it might be a good idea to have that separate